newIDE/app/src/EventsSheet/EventsTree/Renderers/CommentEvent.js

```diff
@@ -29,7 +28,6 @@ const styles = {
     flexWrap: 'wrap',
     padding: 5,
     overflow: 'hidden',
-    minHeight: 35,
```

Comment:
Just thought about it, might be worth checking if a comment without any text is shown properly or not.

Reply:
Nope, you're right, i'll look into it